### PR TITLE
Adding examples back to OpenCensus repo.

### PR DESCRIPTION
**Pull request by Vizerai**
References and build file have been fixed to work with new setup where the filter lies in the grpc repo.

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -31,7 +31,7 @@ http_archive(
 # gRPC
 http_archive(
     name = "com_github_grpc_grpc",
-    urls = ["https://github.com/grpc/grpc/archive/master.tar.gz"],
+    urls = ["https://github.com/grpc/grpc/archive/master.zip"],
```
Comment (g-easy) [inline WORKSPACE]: What's happening here?

File: examples/grpc/hello_client.cc [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+#include <iostream>
+#include <memory>
+#include <string>
+
+#include <grpcpp/grpcpp.h>
+
+#include "examples/grpc/hello.grpc.pb.h"
+#include "examples/grpc/hello.pb.h"
+#include "examples/grpc/stackdriver.h"
+#include "include/grpcpp/opencensus.h"
```
Comment (g-easy) [inline examples/grpc/hello_client.cc]: Can this be `#include <grpcpp/opencensus.h>` with the angle brackets?

File: examples/grpc/hello_server.cc [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+#include <string>
+
+#include <grpcpp/grpcpp.h>
+
+#include "absl/strings/numbers.h"
+#include "absl/strings/str_cat.h"
+#include "examples/grpc/hello.grpc.pb.h"
+#include "examples/grpc/hello.pb.h"
+#include "examples/grpc/stackdriver.h"
+#include "include/grpcpp/opencensus.h"
```
Comment (g-easy) [inline examples/grpc/hello_server.cc]: Ditto.

File: examples/grpc/hello_server.cc [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+#include "opencensus/exporters/stats/stackdriver/stackdriver_exporter.h"
+#include "opencensus/exporters/stats/stdout/stdout_exporter.h"
+#include "opencensus/exporters/trace/stackdriver/stackdriver_exporter.h"
+#include "opencensus/exporters/trace/stdout/stdout_exporter.h"
+#include "opencensus/stats/stats.h"
+#include "opencensus/trace/sampler.h"
+#include "opencensus/trace/span.h"
+#include "opencensus/trace/trace_config.h"
+#include "prometheus/exposer.h"
+#include "src/cpp/ext/filters/census/grpc_plugin.h"
```
Comment (g-easy) [inline examples/grpc/hello_server.cc]: Is this needed? Is this an internal header?

File: examples/grpc/hello_client.cc [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+namespace {
+
+using examples::HelloReply;
+using examples::HelloRequest;
+using examples::HelloService;
+
+}  // namespace
+
+int main(int argc, char **argv) {
```
Comment (g-easy) [inline examples/grpc/hello_client.cc]: Why did the stars move? (char**_ -> char_**)

Comment (Vizerai) [reply examples/grpc/hello_client.cc]: clang-format moved them.

Comment (g-easy) [reply examples/grpc/hello_client.cc]: You're right, `clang-format-3.9` puts the space first. Sorry for the noise.

File: WORKSPACE (hunk 1 above)
Comment (Vizerai) [reply WORKSPACE]: Reverted.

File: examples/grpc/hello_client.cc (hunk 2 above)
Comment (Vizerai) [reply examples/grpc/hello_client.cc]: Done.

File: examples/grpc/hello_server.cc (hunk 3 above)
Comment (Vizerai) [reply examples/grpc/hello_server.cc]: Done.

File: examples/grpc/hello_server.cc (hunk 4 above)
Comment (Vizerai) [reply examples/grpc/hello_server.cc]: Yes, this is needed for calls to GetSpanFromServerContext.

**Pull request merged**